src/Microsoft.VisualStudio.Services.Agent/Util/VarUtil.cs

```diff
@@ -1,12 +1,48 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
 
 namespace Microsoft.VisualStudio.Services.Agent.Util
 {
-    public static class VarUtil
+    [ServiceLocator(Default = typeof(VarUtil))]
+    public interface IVarUtil : IAgentService
     {
+        void PrependPath(string directory);
+        void SetEnvironmentVariable(string name, string value);
+    }
+
+    public sealed class VarUtil : AgentService, IVarUtil
+    {
+        public void PrependPath(string directory)
```

Comment:
what about remove the interface, just make all methods static?
currently, only PrependPath() and SetEnvironmentVariable() are interface methods, all others are static methods.